pkg/kev/config/svcext.go

```diff
@@ -260,18 +267,18 @@ func PodSecurityWithDefaults() PodSecurity {
 	}
 }
 
-func ServiceTypeFromCompose(svc *composego.ServiceConfig) (string, error) {
-	serviceType := NoService
+func ServiceTypeFromCompose(svc *composego.ServiceConfig) (ServiceType, error) {
+	var candidate = "none"
```

Comment:
What's the reason for this change (and the other cases below)?

Reply:
I've reverted to using the raw values determined by the various rules.

Then, `inferServiceTypeFromComposeValue` can use its own internal rules to turn the candidate raw value into the actual `ServiceType` enum.

Otherwise, it can get confusing as to why I'm using an enum to start with. Changing it to a string, and then `inferServiceTypeFromComposeValue` changing it back to an enum.